frontend/src/pages/ConfigurationPage.module.scss

```diff
@@ -1,14 +1,43 @@
 @import '../shared.scss';
 
-.configurationPage {
+.settingsPage {
   @extend .layoutCol;
-  margin-left: 27px;
-  margin-top: 20px;
-  gap: 20px;
+  margin: 10px 20px;
+  gap: 80px;
 }
 
-.columnHeader {
-  @extend .typography-h3;
```

Comment:
`IntegrationConfiguration.tsx` uses this class, and those seem to have lost the h3 styling.

If this is intentional, should remove the uses of this class. Otherwise, it could be added back (or moved to `IntegrationConfiguration.module.scss`)